### PR TITLE
Migrate to tldr for tests

### PR DESCRIPTION
**Pull request by searls**
Trying to harden [tldr](https://github.com/tendersearls/tldr) and so I took a stab at swapping it here. Thoughts?  (Locally, test suite time went from 0.375s => 0.296s for me)

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-  t.libs << "test"
-  t.warning = true
-  t.verbose = true
-  t.test_files = FileList["test/**/*_test.rb"]
-end
-
 FILES = [ "lib/tinygql/nodes.rb", "lib/tinygql/visitors.rb" ]
 
-task :test => FILES
+task :test => [*FILES, :tldr]
```
Comment (tenderlove) [inline Rakefile:7]: I don't think this is right.  I think it should just be `task :tldr => FILES`

**Review by tenderlove — COMMENTED** [review]
Does the rake task run with `-w` enabled on Ruby?

**Comment by searls** [comment]
Effectively, yes. Right now TLDR sets $VERBOSE=true unless --no-warnings is set.

File: Rakefile (hunk 1 above)
Comment (searls) [reply Rakefile:7]: That's fine. I just didn't want to go in and change the testing command `bundle exec rake test` in CI to `tldr` without your OK

**Pull request merged**